### ISSUE TITLE
Request to optimize the display on high DPI display

### ISSUE THREAD

**Issue by ZhangShuTHU** [kanors-emr/Veda2.0-Installation#3]
In 4K or higher resolution screen, the font  is fuzzy, the interface layout is disordered, I think the related UI design needs to be improved.

**Comment by jamesglynn** [comment]
My Screen is 4K and Veda2_1.205.1.1 looks and performs as expected

**Comment by ZhangShuTHU** [comment]
Thank you, James! After I downloaded the latest version of 1.219.1.1, the window layout is normal, but the font is still a little jagged compared with the text in the browser. I tried the options in Windows compatibility settings. If "application" is selected, the font is clear, but the layout is chaotic. If "system" or "system (enhanced)" is selected, the layout is normal and the font is fuzzy. I don't know if this is the reason of my computer setting or the limitation of software?
DPI setting in "System settings-display" to 200% or 150% will cause problems. It is normal only when it is set to 100%.

![image](https://user-images.githubusercontent.com/38218229/109001688-304c7100-76e0-11eb-97f6-ca67481a10f3.png)
